### ISSUE TITLE
Validation of "ietf-netconf-notifications@2012-02-06" when: XPath location step 'confirm-event' (at 4) selects zero nodes

### ISSUE THREAD

**Issue by fooljohnny** [YangModels/yang#364]
https://github.com/YangModels/yang/blob/e599022cbbffb5f9e7b30cdfc9bd97ec680c17a0/standard/ietf/RFC/ietf-netconf-notifications%402012-02-06.yang#L286

load by netconf browser or validate by confd, both report warning :
![image](https://user-images.githubusercontent.com/10184525/36657844-16f0c87c-1b09-11e8-8ce6-1c77cc3101d5.png)

the right xpath should be : when "./confirm-event != 'timeout'", right?

**Comment by einarnn** [comment]
Yes, I think it should be. I'll try and follow up with the model authors.

Cheers,

Einar

**Comment by einarnn** [comment]
Already a known issue:

https://www.rfc-editor.org/errata_search.php?rfc=6470

Cheers,

Einar

**Comment by fooljohnny** [comment]
aha，thanks a lot.
